eng/common/TestResources/New-TestResources.ps1

```diff
@@ -202,7 +201,7 @@ try {
         $context = Get-AzContext;
         if (!$context) {
             Log "You are not logged in; connecting to 'Azure SDK Developer Playground'"
```

Comment:
This message is not correct anymore.

Reply:
Thanks. I missed that, but not really worth starting this process over. I opened https://github.com/Azure/azure-sdk-tools/issues/1407 to track. Maybe we could use a small look-up table of subscription IDs to use a friendly name when possible.